cmd/cluster-agent-cloudfoundry/subcommands/run/command.go

```diff
@@ -83,8 +83,7 @@ func Commands(globalParams *command.GlobalParams) []*cobra.Command {
 					LogParams:    log.ForDaemon(command.LoggerName, "log_file", path.DefaultDCALogFile),
 				}),
 				core.Bundle(),
-				forwarder.Bundle(),
-				fx.Provide(defaultforwarder.NewParamsWithResolvers),
+				forwarder.BundleWithProvider(defaultforwarder.NewParamsWithResolvers),
```

Comment:
Should we use `forwarder.Bundle(defaultforwarder.NewParamsWithResolvers),` ?

Reply:
The code will not compile as the type of the argument for `forwarder. Bundle` is `defaultforwarder.Params` whereas the type of `NewParamsWithResolvers` is `func (config.Component, log log.Component) Params`